src/test/java/UserServiceTests.java

```diff
@@ -0,0 +1,149 @@
+import Models.Group;
+import Services.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserServiceTests {
+
+    @BeforeEach
+    public void clear() {
+        UserService.clearBase();
+    }
+
+    @Test
+    public void testUserAuthentication() {
+        Assertions.assertTrue(UserService.register("user1", "password"));
+        Assertions.assertFalse(UserService.register("user1", "password"));
+        Assertions.assertTrue(UserService.authenticate("user1", "password"));
+        Assertions.assertFalse(UserService.authenticate("user2", "password"));
+    }
+
+    @Test
+    public void testUserAuthorization() {
```

Comment:
In most cases, such tests should be split into several. Otherwise, that specific test can fail for a number of reasons at once.

Reply:
Split all tests into smaller chunks:)